extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php

```diff
@@ -62,8 +73,10 @@ public function index() {
 		$this->isUserLoggedIn = $wgUser->isLoggedIn();
 
 		// remove wikia plus for now for all languages
+		// Harnash: Not needed?
```

Comment:
can we solve this doubt before merging this PR or add a TODO with a follow up SUS ticket to investigate this?